dhis-2/dhis-support/dhis-support-jdbc/src/main/java/org/hisp/dhis/jdbc/batchhandler/ReservedValueBatchHandler.java

```diff
@@ -118,19 +118,19 @@ public ReservedValue mapRow( ResultSet resultSet )
     {
         Calendar expires = Calendar.getInstance();
 
-        ReservedValue rv = new ReservedValue();
+        ReservedValue.ReservedValueBuilder rv = ReservedValue.builder();
 
         expires.setTime( resultSet.getDate( "expirydate" ) );
```

Comment:
I don't get the point of havinga Calendar for this.
Can't you just 
`
rv.expiryDate( resultSet.getDate( "expiryDate") );
`
without creating a Calendar instance ?

Just as a style note: in my opinion, this would be more readable with methods chain like:

```java
    @Override
    public ReservedValue mapRow( ResultSet resultSet )
        throws SQLException
    {
        return ReservedValue.builder()
            .id( resultSet.getInt( "reservedvalueid" ) )
            .ownerObject( resultSet.getString( "ownerobject" ) )
            .ownerUid( resultSet.getString( "ownerUid" ) )
            .key( resultSet.getString( "key" ) )
            .value( resultSet.getString( "value" ) )
            .expiryDate( resultSet.getDate( "expiryDate" ) )
            .created( resultSet.getDate( "created" ) )
            .build();
    }

```

Reply:
thanks i'll have a look i had to refactor many things ...